cross-chain-atomic-swap-cordapp/src/main/kotlin/jp/co/layerx/cordage/crosschainatomicswap/Utils.kt

```diff
@@ -25,16 +25,16 @@ private val ethAddress = Properties()
 
 fun Party.ethAddress(): String {
     if (ethAddress.isEmpty) {
-        val url = javaClass.classLoader.getResource("ethAddress.properties")
-        if (url != null) {
-            ethAddress.load(url.openStream())
-        }
         val root = System.getProperty("user.dir")
         val path = Paths.get("$root/cordapps/config/cross-chain-atomic-swap-cordapp-0.1.conf")
         if (Files.isReadable(path)) {
             ethAddress.load(Files.newInputStream(path))
+        } else {
+            val path = Paths.get("$root/src/test/resources/config.conf")
```

Comment:
I'm not sure but is this necessary? It seems that this config file has been called from the `TestUtils`

Reply:
Utils.kt is called inside the cordapp itself (when you do party.ethAddress()) for example in `SetttleAtomicSwap.kt` : 

````kotlin
            "Proposer's address must equal to fromEthereumAddress." using (inputProposal.fromEthereumAddress == inputProposal.proposer.ethAddress())
````

And if I don't add this line, then it looks only for `$root/cordapps/config/cross-chain-atomic-swap-cordapp-0.1.conf` which does not exist and load nothing .

I agree it's not optimal :)

Reply:
Oh, I totally got it!

Reply:
In this case, I think it is good to use `javaClass.classLoader.getResource("config.conf")` to get a path because it finds a file path in the resources directory automatically.

Deleted lines 28-31 can be a good reference.